scripts/sct_register_to_template.py

```diff
@@ -274,7 +292,7 @@ def main(args=None):
     labels = check_labels(fname_landmarks, label_type=label_type)
 
     vertebral_alignment = False
```

Comment:
Would it make sense to rename this variable `label_alignment`?

Reply:
yup! it would definitely would. I would call it call it `level_alignment` though